rdbms-sprint/data/migrations/20181012111910_projects.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('projects', function(projects) {
+      projects.increments();
+  
+      projects.string('name', 128).notNullable();
+      projects.text('description').notNullable();
+      projects.boolean('completed').defaultTo(false);
```

Comment:
Note that `defaultTo` is not supported in sqlite3, but it's good to use in case you switch database engines.